src/zzz_od/operation/back_to_normal_world.py

```
@@ -91,7 +92,7 @@ def check_screen_and_run(self) -> OperationRoundResult:
             return self.round_success('脱离卡死', wait=1)
 
         # 通用返回按钮（识别点击型）
-        # 需要在"完成"前面，某些插件场景可能会识别到'返回'和"完成"同时存在
+        # 需要在"完成"前面，否则在丽都城募app（若购买了大月卡）会误点击“已完成购买” issue #2005
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)